core/config_test.go

```diff
@@ -256,40 +285,45 @@ func TestChainConfig_GetFeature(t *testing.T) {
 	}
 }
 
-var allAvailableConfigKeys = []string{
+var allAvailableDefaultConfigKeys = []string{
+	"difficulty",
+	"gastable",
+	"eip155",
+}
+var allAvailableTestnetConfigKeys = []string{
 	"difficulty",
 	"gastable",
 	"eip155",
+	"reward",
+}
+var unavailableConfigKeys = []string{
+	"foo",
+	"bar",
+	"monkey",
 }
 
 // TestChainConfig_EventuallyGetAllPossibleFeatures should aggregate all available features from previous branches
 func TestChainConfig_GetFeature2_EventuallyGetAllPossibleFeatures(t *testing.T) {
-	c := makeTestChainConfig()
-	for _, id := range allAvailableConfigKeys {
-		if _, _, ok := c.GetFeature(big.NewInt(5000000), id); !ok {
+	c := getDefaultChainConfigSorted()
+	for _, id := range allAvailableDefaultConfigKeys {
+		if _, _, ok := c.GetFeature(big.NewInt(50000000), id); !ok {
```

Comment:
The zeros don't match up, in this arg there are 7 zeros, the test seems to require 6 zeros.